OmgvaPOS/src/UserManagement/Repository/UserRepository.cs

```diff
@@ -97,7 +98,7 @@ public List<EmployeeSchedule> GetUserSchedules(long id)
                 throw new ApplicationException("Error retrieving user schedules.", ex);
             }
         }
-        public List<Order> GetUserOrders(long id)
+        public List<Order.Entities.OrderEntity> GetUserOrders(long id)
```

Comment:
Why can't you just write OrderEntity? This is very verbose.

Reply:
My bad, this was due to restructuring and happened automatically, tried to fix most of these, thanks for noticing